src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.cs

```diff
@@ -90,13 +90,13 @@ private static IEnumerable<T> Prioritize<T>(IEnumerable<T> items, Func<T, bool>
     /// </summary>
     private static Task PerformParallelSearchAsync<T>(
```

Comment:
>PerformParallelSearchAsync [](http://example.com/codeflow?start=24&length=26)

Curious if this method provides any value now.

Reply:
I can consider removing in follow up